### PR TITLE
Update dependencies to `polkadot-v0.9.9` series

### PR DESCRIPTION
**Pull request by JoshOrndorff**
This PR updates the dependencies and the Rust toolchain.

This unblocks #26 which I'll also tackle now.

### CODE REVIEW

**Comment by JoshOrndorff** [comment]
So the benchmarks seem not to be working / building. I can reproduce the error from the CI with `cargo test --features runtime-benchmar` but I'm not sure what I did to break that or how to fix it.

**Comment by girazoki** [comment]
This branch now fails in the fmt too. So far I have been able to figure out that:
- The failure happens because we dont have a lock file
- The failure happens due to some discrepancy in the cumulus-parachain-system or cumulus-primitives-parachain-inherent.  #43 and #44 should be able to fix that, so I might try to start from #44 to fix this

**Review by girazoki — APPROVED** [review]

**Comment by girazoki** [comment]
This can be merged now

File: rust-toolchain [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [toolchain]
-channel = "nightly-2021-02-21"
+channel = "nightly-2021-08-01"
```
Comment (notlesh) [inline rust-toolchain:2]: Is this intentional? I don't see a corresponding bump in https://github.com/PureStake/moonbeam/pull/740/ -- do we want to keep them in sync?

Comment (girazoki) [reply rust-toolchain:2]: IT's already in master https://github.com/PureStake/moonbeam/blob/master/rust-toolchain

**Review by notlesh — APPROVED** [review]
Looks good. I had a quick question that was worth reviewing.

Also, we may want to re-run benchmarks after our deps update. I can do that, just let me know when.

**Pull request merged**